.circleci/docker/Dockerfile.ubuntu1904

```diff
@@ -0,0 +1,117 @@
+# vim:syntax=dockerfile
+#------------------------------------------------------------------------------
+# Dockerfile for building and testing Solidity Compiler on CI
+# Target: Ubuntu 18.10 (Cosmic Cuttlefish)
+#
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2016-2019 solidity contributors.
+#------------------------------------------------------------------------------
+FROM buildpack-deps:disco
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN set -ex; \
+	apt-get update; \
+	apt-get install -qqy --no-install-recommends \
+		build-essential \
+		software-properties-common \
+		cmake ninja-build clang++-7 \
+		libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev \
+		libboost-program-options-dev \
+		libjsoncpp-dev \
+		llvm-7-dev libcvc4-dev libleveldb1d \
+		; \
+	apt-get install -qy python-pip python-sphinx; \
+	update-alternatives --install /usr/bin/llvm-symbolizer llvm-symbolizer /usr/bin/llvm-symbolizer-7 1; \
+	pip install codecov; \
+	rm -rf /var/lib/apt/lists/*
+
+RUN set -ex; \
+	git clone --depth=1 https://github.com/Z3Prover/z3.git /usr/src/z3; \
+	mkdir /usr/src/z3/build; \
+	cd /usr/src/z3/build; \
+	cmake -DCMAKE_BUILD_TYPE="Release" -DCMAKE_INSTALL_PREFIX="/usr" -G "Ninja" ..; \
+	ninja; \
+	ninja install/strip; \
+	rm -rf /usr/src/z3
+
+ARG ALETH_VERSION="1.6.0"
```

Comment:
Is the documentation updated that talks about where to modify the aleth version?
What about verifying the hash?

Reply:
The aleth version didn't change. It's a copy'n'paste from `scripts/tests.sh` and also mentioned in `/docs/contributing.rst`. -- but the hash stuff I've just added. thanks :)

Reply:
I meant that we have to mention somewhere that this is one of the many places where the version has to be changed.